stubs/Reflection.phpstub

```diff
@@ -76,6 +76,18 @@ class ReflectionProperty implements Reflector
      * @return array<ReflectionAttribute<TClass>>
      */
     public function getAttributes(?string $name = null, int $flags = 0): array {}
+
+    /**
+     * @since 7.4
+     * @psalm-assert-if-true ReflectionType $this->getType()
+     */
+    public function hasType() : bool {}
+
+    /**
+     * @since 7.4
+     * @psalm-mutation-free
```

Comment:
@caugner I think it works with the `@psalm-mutation-free` you said 👍  (apart from adding the `7.4` check)

Reply:
Just to check: Did you need to add mutation-free to make the test pass? So marking the test with 7.4 was not enough?

Reply:
Yes, without: https://psalm.dev/r/5d10995beb
with: https://psalm.dev/r/8e9b30898d

Reply:
Thanks, glad to hear it works! This reminds me that I should use that annotation in my own projects as well. 🙂